end_app/accounts/views.py

```diff
@@ -0,0 +1,54 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.decorators import login_required
+from .forms import LoginForm, UserRegistrationForms
+
+
+# Create your views here.
```

Comment:
tego się pozbądźmy

Reply:
#5 "from .forms import LoginForm, UserRegistrationForms" - Nie mogę się pozbyć tego importa ponieważ jest on wykorzystywany w user_login oraz register
#8  # Create your views here. - Usunięto przeoczony komentarz